packages/editor-ui/src/composables/useStyles.ts

```diff
@@ -0,0 +1,31 @@
+const APP_Z_INDEXES = {
```

Comment:
This is so much better than random numbers spread across the codebase. Thanks for addressing this 🙇🏽